### PR TITLE
🐛 Force File Picker progress to be between 1 and 99

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Buggy scenario: With an overloaded backend, the multipart uploading process was very slow, making the buggy File Picker's progress calculation negative for the very first uploaded parts and, therefore, study's validation faulty. This PR forces the value to be between 1 (started) and 99 (almost finished).

### CODE REVIEW

**Review by mrnicegyu11 — APPROVED** [review]
Restoring a bug free frontend < 3

**Review by sanderegg — APPROVED** [review]
ahah. was -1 a bad thing?

**Comment by odeimaiz** [comment]
> ahah. was -1 a bad thing?

Bad enough for making it hotfix candidate...

**Review by pcrespov — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/file/FilePicker.js [hunk 1]
```diff
@@ -658,7 +658,9 @@ qx.Class.define("osparc.file.FilePicker", {
               this.__uploadedParts[chunkIdx]["e_tag"] = eTag.slice(1, -1);
               const uploadedParts = this.__uploadedParts.filter(uploadedPart => uploadedPart["e_tag"] !== null).length;
               const progress = uploadedParts/this.__uploadedParts.length;
-              this.getNode().getStatus().setProgress(100*progress-1);
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/file/FilePicker.js:661]: Q: When you describe that you get displayed negative numbers, isn't it what happens when the progress is actually 0? 
i.e. ``progress=0 -> 100*0-1 = -1`` and when the upload finished ``progress=1 -> 100*1-1 = 99``

wouldn't it be solved using a linear function as `` 98*progress +1`` ??

**Review by GitHK — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/file/FilePicker.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/file/FilePicker.js:661]: Correct, that's yet another solution.

**Pull request merged**